src/UBAFeeCalculator/UBAFeeSpokeCalculatorAnalog.ts

```diff
@@ -138,12 +138,18 @@ export function getEventFee(
 
   // Next, we'll need to compute the first balancing fee from the running balance of the spoke
   // to the running balance of the spoke + the amount
-  const balancingFee = computePiecewiseLinearFunction(
+  let balancingFee = computePiecewiseLinearFunction(
     flowCurve,
     runningBalance,
     amount.mul(flowType === "inflow" ? 1 : -1)
   );
 
+  // We need to account for the balancing fee being greater than the incentive balance
+  if (balancingFee.gt(incentiveBalance)) {
+    const discountFactor = min(BigNumber.from(1), balancingFee.sub(incentiveBalance).div(balancingFee));
```

Comment:
```suggestion
    const discountFactor = min(BigNumber.from(1), (balancingFee.sub(incentiveBalance)).div(balancingFee));
```

Reply:
The current code satisfies the order of operations. VSCode removes the suggestion as redundant if I include it.

Reply:
does it?